js/bibox.js

```diff
@@ -1274,6 +1346,96 @@ module.exports = class bibox extends Exchange {
         };
     }
 
+    async transfer (code, amount, fromAccount, toAccount, params = {}) {
+        await this.loadMarkets ();
+        const currency = this.currency (code);
+        let method = undefined;
+        const request = {};
+        if (fromAccount === 'wallet' && toAccount === 'spot') {
+            method = 'privatePostAssetsTransferSpot';
+            request['type'] = 0;
+        } else if (fromAccount === 'spot' && toAccount === 'wallet') {
+            method = 'privatePostAssetsTransferSpot';
+            request['type'] = 1;
+        } else if (fromAccount === 'wallet' && toAccount === 'margin') {
+            method = 'privatePostCreditTransferAssetsBase2credit';
+        } else if (fromAccount === 'margin' && toAccount === 'wallet') {
+            method = 'privatePostCreditTransferAssetsCredit2base';
+        }
+        if (method === 'privatePostAssetsTransferSpot') {
+            request['symbol'] = currency['id'];
+            request['amount'] = this.currencyToPrecision (code, amount);
+        } else if (method === 'privatePostCreditTransferAssetsBase2credit' || method === 'privatePostCreditTransferAssetsCredit2base') {
+            request['coin_symbol'] = currency['id'];
+            request['amount'] = amount;
+            const symbol = this.safeString (params, 'symbol');
+            params = this.omit (params, 'symbol');
+            if (symbol === undefined) {
+                throw new ExchangeError ('to transfer to and from margin you must specify the `symbol` of the market for fixed margin');
+            }
+            const market = this.market (symbol);
+            request['pair'] = market['id'];
```

Comment:
Even though in the docs the parameter `pair` and `amount` appear as optional, the request fail without them.

Reply:
Can address in seperate PR when adding transfer function